src/util/system.cpp

```diff
@@ -1130,11 +1130,13 @@ void runCommand(const std::string& strCommand)
     if (strCommand.empty()) return;
 #ifndef WIN32
     int nErr = ::system(strCommand.c_str());
-#else
-    int nErr = ::_wsystem(std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t>().from_bytes(strCommand).c_str());
-#endif
     if (nErr)
         LogPrintf("runCommand error: system(%s) returned %d\n", strCommand, nErr);
+#else
+    // Second argument "verb" (=action to be performed) set to NULL results in action "open"
+    if (!ShellExecute(NULL, NULL, strCommand.c_str(), NULL, NULL, SW_HIDE))
```

Comment:
Does this actually work at all? From the docs I would expect it doesn't.

Reply:
Yes it does, I tested it before comitting.

Reply:
Actually you are right about the condition I messed up the return values of [ShellExecuteEx](https://docs.microsoft.com/en-us/windows/desktop/api/Shellapi/nf-shellapi-shellexecuteexa) (boolean) and [ShellExecute](https://docs.microsoft.com/en-us/windows/desktop/api/shellapi/nf-shellapi-shellexecutea#return-value) (HINSTANCE). I am going to fix it right now.